src/Cake.Core/Tooling/ToolResolutionStrategy.cs

```diff
@@ -130,6 +136,9 @@ private FilePath LookInPath(string tool)
                     }
                 }
 
+                var allPaths = string.Join(",", _path);
+                _log.Verbose($"Could not resolve path for tool \"{tool}\" using these directories:");
```

Comment:
I would prefer this to be diagnostic output.

Reply:
Fixed.